src/trio/_channel.py

```diff
@@ -299,8 +299,8 @@ def statistics(self) -> MemoryChannelStats:
         return self._state.statistics()
 
     def __repr__(self) -> str:
-        return "<receive channel at {:#x}, using buffer at {:#x}>".format(
-            id(self), id(self._state)
+        return (
+            f"<receive channel at {id(self):#x}, using buffer at {id(self._state):#x}>"
```

Comment:
These parens are no longer necessary. I _think_ if we added `id: ruff-format` to `.pre-commit-config.yaml`, it would fix these.

Reply:
Yes, but I think ruff format is intended on replacing black and I think that would be best suited for separate pull request.

Reply:
I tried removing them while preparing ec4c012, but black added them back in.

Reply:
Ok, whatever, black. :P